src/App.jsx

```diff
@@ -28,16 +27,37 @@ class App extends Component {
     }
   };
 
+  async seeDetails(event) {
+    let id = event.target.parentElement.dataset.id;
+    let details;
+    if (id > 0) {
+      details = await axios.get(`/cocktails/${id}`);
+    }
+    this.setState({
+      ingredients: details.data.drink.ingredients,
+      details: details.data.drink
+    });
+  }
+
   render() {
-    let renderResults;
+    let renderResults, renderDetails;
+
     if (Array.isArray(this.state.results) && this.state.results.length > 0) {
       renderResults = (
         <div id="result-list">
           {this.state.results.map(item => {
             return (
-              <div key={item.id}>
-                <h4>{item.name}</h4>
-                {item.category} {item.IBA}
+              <div key={item.id} data-id={item.id}>
+                <br />
+                <div id="drink-name" key={item.id} data-id={item.id}>{item.name}
+                  <button
+                    id="details-button"
+                    onClick={this.seeDetails.bind(this)}
+                    key={item.id}
+                  >
+                    Details
+                </button></div>
+                {item.category} {item.IBA} <br />
```

Comment:
I would extract this to a separate component but it can be done as a refactoring ticket

Reply:
Yes, we are planning to refactor everything in a separate PR to clean it up after we get the main functionality built out.